server/views/pages/calculation/checkInformation.njk

```diff
@@ -31,8 +32,12 @@
 {% block content %}
     {% set validationErrors = model.validationErrors %}
     {% include "../partials/customErrorBanner.njk" %}
+    {% include "../partials/ersedRecallNotificationBanner.njk" %}
```

Comment:
I think this notification banner is on the wrong page. It says in the ticket it should be on the release dates page (Calculation summary)

Reply:
Done.